Mathlib/Data/Real/ConjExponents.lean

```diff
@@ -220,3 +226,120 @@ protected lemma Real.IsConjExponent.toNNReal {p q : ℝ} (hpq : p.IsConjExponent
   one_lt := by simpa using hpq.one_lt
   inv_add_inv_conj := by rw [← toNNReal_inv, ← toNNReal_inv, ← toNNReal_add hpq.inv_nonneg
     hpq.symm.inv_nonneg, hpq.inv_add_inv_conj, toNNReal_one]
+
+namespace ENNReal
+
+/-- Two nonnegative real exponents `p, q` are conjugate if they are `> 1` and satisfy the equality
+`1/p + 1/q = 1`. This condition shows up in many theorems in analysis, notably related to `L^p`
+norms. -/
+@[mk_iff]
+structure IsConjExponent (p q : ℝ≥0∞) : Prop where
+  inv_add_inv_conj : p⁻¹ + q⁻¹ = 1
+
+/-- The conjugate exponent of `p` is `q = p/(p - 1)`, so that `1/p + 1/q = 1`. -/
```

Comment:
```suggestion
/-- The conjugate exponent of `p` is `q = 1 + (p - 1)⁻¹`, so that `1/p + 1/q = 1`. -/
```
I'd prefer to have the docstring and the definition line up here - this isn't a case where the actual definition is sufficiently complicated that lying is better. I'm guessing that p/(p-1) doesn't work as the definition because of ENNReal arithmetic being weird?

Reply:
Yep indeed. ENNReal arithmetic is weird: Try inputting ` p = infty`